Mathlib/LinearAlgebra/BilinearForm/Hom.lean

```diff
@@ -83,9 +83,9 @@ variable (R₂)
 /-- The linear map obtained from a `BilinForm` by fixing the left co-ordinate and evaluating in
 the right.
 This is the most general version of the construction; it is `R₂`-linear for some distinguished
-commutative subsemiring `R₂` of the scalar ring.  Over a semiring with no particular distinguished
-such subsemiring, use `toLin'`, which is `ℕ`-linear.  Over a commutative semiring, use `toLin`,
-which is linear. -/
+commutative subsemiring `R₂` of the scalar CommRing.  Over a semiring with no particular
+distinguished such subsemiring, use `toLin'`, which is `ℕ`-linear.  Over a commutative semiring,
+use `toLin`, which is linear. -/
 def toLinHom : BilinForm R M →ₗ[R₂] M →ₗ[R₂] M →ₗ[R] R where
```

Comment:
I think `R₂` here can be replaced with `R` now.

Reply:
Done. Should I replace `R₂` with `R` elsewhere?

Reply:
Presumably the same applies to `flipHom` etc in `LinearAlgebra/BilinearForm/Basic`?

Reply:
@eric-wieser  I've removed  `R₂`/`M₂` and `R₃`/`M₃` from other locations as well - not sure if that was what you wanted at this stage? You can always revert the commits if not.